typescript/best-practice-web/test/specs/visual.spec.ts

```diff
@@ -0,0 +1,20 @@
+import loginPage from '../pages/login.page';
+import {LOGIN_USERS} from '../e2eConstants'
+import cartPage from '../pages/cart.page';
+import inventoryPage from '../pages/inventory.page';
+
+describe('My web app', () => {
+    it('should look correct on desktop web', () => {
```

Comment:
This is not an atomic and autonomous test case, so not a best practice. You are testing the login , the inventory and the cart in 1 test. Is that on purpose?

Reply:
@wswebcreation this is the best practice for visual testing so that the build appears correctly. I'm not sure if there's a better way to do it. Furthermore, a visual failure always requires manual intervention so there is no gain to split this into individual tests as any failure triggers the same response. Different from functional testing which triggers a different action depending on the failure. Open to other suggestions though

<img width="1122" alt="Screen Shot 2021-05-10 at 3 47 30 PM" src="https://user-images.githubusercontent.com/2136245/117716008-0b2bb280-b1a7-11eb-9005-bc5011062e8a.png">